.github/workflows/presto-kerberos-integration-test.yml

```diff
@@ -0,0 +1,103 @@
+name: Kerberized Presto Integration Test
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'release**'
+      - 'feature**'
+    tags:
+      - '**'
+    paths:
+    - '**/presto_jdbc/**'
+    - '**/presto_jdbc.clj'
+
+jobs:
+
+  run-presto-kerberos-test:
+    # TODO: build some custom Docker image that pre-installs openjdk, babashka, docker-compose, etc.
+    runs-on: ubuntu-20.04
+    timeout-minutes: 40
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Check out Presto Kerberos Docker Compose
+        uses: actions/checkout@master
+        with:
+          # TODO: fork this to metabase if everything works out
+          repository: mik-laj/presto-kerberos-docker
+          token: ${{ secrets.GITHUB_TOKEN }}
+          path: presto-kerberos-docker
+      - name: List files
+        run: pwd && ls -1 presto-kerberos-docker
+      # TODO: switch to doing this? https://github.com/nektos/act/issues/112#issuecomment-803619401
+      - run: >
+          sudo curl -L https://github.com/docker/compose/releases/download/1.29.2/docker-compose-`uname -s`-`uname -m`
+          -o /usr/local/bin/docker-compose
+      - run: sudo chmod +x /usr/local/bin/docker-compose
+      - name: Bring up Presto+Kerberos cluster
+        run: cd presto-kerberos-docker && docker-compose up -d
+      - name: Run Presto test query from command line
+        run: cd presto-kerberos-docker && ./test.sh
+      - name: Install OpenJDK 8
+        run: sudo apt update && sudo apt-get -y install openjdk-8-jdk
+      - name: Capture Presto server CA
+        run: >
+          openssl s_client -connect localhost:37778 2>/dev/null </dev/null
+          |  sed -ne '/-BEGIN CERTIFICATE-/,/-END CERTIFICATE-/p' > resources/presto-ssl-root-ca.pem
+      - name: Convert root CA from PEM to DER
+        run: >
+          openssl x509 -outform der -in resources/presto-ssl-root-ca.pem
+          -out resources/presto-ssl-root-ca.der
+      - name: Copy OpenJDK 8 root truststore to resources
+        run: >
+          cp /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/security/cacerts resources/cacerts-with-presto-ca.jks
+      - name: Copy Presto SSL keystore to resources
+        run: docker cp presto-kerberos:/tmp/ssl_keystore.jks resources
+      - name: Copy krb5.conf file to resources
+        run: docker cp presto-kerberos:/etc/krb5.conf resources
+      - name: Copy client.keytab file to resources
+        run: docker cp presto-kerberos:/home/presto/client.keytab resources
+      - name: Add Presto root CA to truststore
+        run: >
+          keytool -noprompt -import -alias presto-kerberos -keystore resources/cacerts-with-presto-ca.jks
+          -storepass changeit -file resources/presto-ssl-root-ca.der -trustcacerts
+      - name: Clone MBA
+        uses: actions/checkout@master
+        with:
+          repository: metabase/mba
+          ref: master
```

Comment:
I'd pin this to a sha till mba is stable enough

Reply:
This is all taken care of now (integration test running against `mba` master).